app/scripts/background.js

```diff
@@ -919,10 +919,9 @@ async function onInstall() {
   const storeAlreadyExisted = Boolean(await localStore.get());
   // If the store doesn't exist, then this is the first time running this script,
   // and is therefore an install
-  if (
-    !storeAlreadyExisted &&
-    !(process.env.METAMASK_DEBUG || process.env.IN_TEST)
-  ) {
+  if (process.env.IN_TEST) {
```

Comment:
should we add `!storeAlreadyExisted` here？

Reply:
No, we cannot add `!storeAlreadyExisted` here, i tested, in testing mode, the `storeAlreadyExisted` is always true. Because we have another line of code before `const localStore = inTest ? new ReadOnlyNetworkStore() : new LocalStore();`, it will initialize the local storage when we are in testing mode, and it makes storeAlreadyExisted always true in testing mode. I also tried to change it to `const localStore = new LocalStore();`, then all the e2e tests fail because the local storage is not initialized and they cannot login.

Reply:
Gotcha!